app/models/search_builder.rb

```diff
@@ -34,7 +34,7 @@ def readable_states
   end
 
   def models_to_solr_clause
-    [ScannedResource, Collection, EphemeraFolder, EphemeraBox, ScannedMap, VectorResource, RasterResource, SimpleResource].join(",")
+    [ScannedResource, Collection, EphemeraFolder, EphemeraBox, ScannedMap, VectorResource, RasterResource, SimpleResource, ArchivalMediaCollection, MediaResource].join(",")
```

Comment:
As this gets longer, I wonder if we should reverse the logic and list models we don't want to show up?

Reply:
Yeah, I think we only want to hide...what, FileSet?

Reply:
I think the ones missing from that list are FileSet, EphemeraProject, EphemraVocabulary, and EphemeraTerm.

Reply:
Do you want me to do that now?

Reply:
Yes, I think it would be good to do now.  If it causes a lot of test failures or something, feel free to make it a separate ticket.

Reply:
Done!